lib/parse.js

```diff
@@ -22,7 +22,11 @@ var parseValues = function parseValues(str, options) {
 
     for (var i = 0; i < parts.length; ++i) {
         var part = parts[i];
-        var pos = part.indexOf(']=') === -1 ? part.indexOf('=') : part.indexOf(']=') + 1;
+
+        var pos = part.indexOf(']=') + 1;
+        if (pos === 0) {
```

Comment:
This isn't the same condition - the original check here was that `pos` was `-1`.